tutors/admin.py

```diff
@@ -1,7 +1,6 @@
 from django.contrib import admin
-from .models import CategoryQuestion, Question, Answer
+from .models import Questions, Answers
```

Comment:
According to [pep-8](https://www.python.org/dev/peps/pep-0008/#imports) a blank line before `.models` is required